### PR TITLE
Checkbox code refactoring

### PR DESCRIPTION
**Pull request by dszczyt**
Your directives can be much lighter. Here is an example. I've added some tests to be sure it does the job. I can help to rewrite your directives like this one. Does anybody have suggestions to improve it ?

### CODE REVIEW

File: dist/angular-semantic-ui.js [hunk 1]
```diff
@@ -144,96 +143,38 @@ angular.module('angularify.semantic.checkbox', [])
...
-          element.children()[0].removeAttribute('checked');
-        }
-      });
-    }
+      vm.toggle = function() { vm.ngModel = !vm.ngModel }
+    },
+    controllerAs: 'vm',
+    bindToController: true,
+    require: [ 'checkbox', 'ngModel' ],
+    template: "<div class=\"ui checkbox\" ng-class=\"vm.classes\">" +
```
Comment (jspdown) [inline dist/angular-semantic-ui.js]: Could you replace double quotes to single ones? I really dislike using backlash character :innocent:

**Comment by jspdown** [comment]
Your code refactoring on this component is definitely better than the previous implementation. I want it merged! 

As mentioned in my previous comment, we prefer single quote for Javascript and double quote for Html.

**Comment by dszczyt** [comment]
OK it's done. Glad you like it !

**Comment by dszczyt** [comment]
FYI, I wanted to keep your original syntax. But I think the "type" and "size" args are not necessary anymore : you could use `<checkbox class="slider large" ng-model="myVar">[label]</checkbox>` to generate a large slider... What do you think about it ?

**Comment by jspdown** [comment]
@dszczyt, `type` and `size` are useless if we use semantic-ui classes. In my mind, angular-semantic-ui should only provide a way to easily use semantic-ui dynamic components on an AngularJs web application. 
So, we should depend on their css classes every time, that was a mistake we did before. I definitely agree with you.

**Comment by dszczyt** [comment]
This is how it looks like. This directive becomes lighter and lighter...

**Comment by jspdown** [comment]
Great! Thanks @dszczyt 
I will try to release a new version on Bower this week.

**Pull request merged**